lib/services/dashd.js

```diff
@@ -2076,25 +2076,31 @@ Dash.prototype.estimateFee = function(blocks, callback) {
  * Will add a transaction to the mempool and relay to connected peers
  * @param {String|Transaction} transaction - The hex string of the transaction
  * @param {Object=} options
- * @param {Boolean=} options.allowAbsurdFees - Enable large fees
+ * @param {Boolean=} options.maxFeeRate - Cap large fees (0 for unlimited)
  * @param {Function} callback
  */
 Dash.prototype.sendTransaction = function(tx, options, callback) {
   var self = this;
-  var allowAbsurdFees = false;
+  var maxFeeRate = 100000; // duff/kB (typically less than 1000)
```

Comment:
The Dash Core RPC help indicates that it uses 0.1 DASH/kB as the default. Might make sense to stick with that :man_shrugging: 
>maxfeerate      (numeric or string, optional, default=0.10) Reject transactions whose fee rate is higher than the specified value, expressed in DASH/kB